competition/track2/train/train.py

```diff
@@ -129,8 +122,12 @@ def train(input_path, output_path):
                     dx = trans_next[0, 0] - trans_cur[0, 0]
                     dy = trans_next[1, 0] - trans_cur[1, 0]
                     dheading = next_heading - current_heading
-                    events = vehicle_data[float(sim_time)]["events"]
-                    if all(value == 0 for value in events.values()):
+                    events = vehicle_data[float(sim_time)].events
+                    attributes = [a for a in dir(events) if not a.startswith('__')]
+                    events_values = []
+                    for a in attributes:
+                        events_values.append(getattr(events, a))
+                    if all(value == 0 for value in events_values):
                         terminal = 0
                     else:
                         terminal = 1
```

Comment:
1) Is this code correctly doing what it is expected to do?
2) Consider simplifying this code.